app/dashboards/blog_post_dashboard.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure there is a requirement for this Dahsboard. I don't think we'll be doing any editing of Blog posts here - it all needs to happen upstream in Wordpress.

Reply:
Removed.